### ISSUE TITLE
Increase memory for the XNAT server

### ISSUE THREAD

**Issue by pipitone** [TIGRLab/admin#472]
Currently `srv-web` is maxed out on ram use and is paging.

**Comment by josephdviviano** [comment]
Yikes!

**Comment by josephdviviano** [comment]
Hey JP I'm going to take this on.

For good measure I'm going to double everything (at least).

**Comment by josephdviviano** [comment]
OK, so,

+ I've updated the ESXI documentation here: https://github.com/TIGRLab/admin/wiki/EXSI. This includes a windows 7 vm we can share to manage kimsrv.
+ I installed `unattended-upgrades` on `srv-web`, cleared out `/boot`, and installed the updated kernels. Hopefully this will address any security issues going forward.
+ I expanded the number of CPU cores available to `srv-web` to 2, with 2 virtual cores each, for a total of 4 virtual cores.
+ I doubled the amount of RAM available to `srv-web` to 8 GB.

We should be good for a while. Fingers crossed!
